rules/mastg-android-unvalidated-deeplink-data.yml

```
@@ -0,0 +1,15 @@
+rules:
+- id: android-unvalidated-deeplink-data
+  languages:
+    - java
+  severity: WARNING
+  metadata:
+    summary: This rule looks for unsecure deeplink.
```

it should be insecure , not unsecure
```suggestion
    summary: This rule looks for insecure deep link configurations.
```